.docker/entrypoint/Gemfile.lock

```diff
@@ -120,7 +120,7 @@ GEM
     pathutil (0.16.2)
       forwardable-extended (~> 2.6)
     public_suffix (4.0.5)
-    rack (2.1.4)
+    rack (1.6.13)
```

Comment:
Codacy found an issue: [Directory traversal in Rack::Directory app bundled with Rack (CVE-2020-8161)](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6058037)